project1/styles.css

```
@@ -0,0 +1,157 @@
+@import url('https://fonts.googleapis.com/css?family=Noto+Sans|PT+Sans|Source+Sans+Pro|Ubuntu&display=swap');
+
+
+/* 
+
+    font-family: 'Source Sans Pro', sans-serif;
+
+    font-family: 'Noto Sans', sans-serif;
+
+    font-family: 'Ubuntu', sans-serif;
+
+    font-family: 'PT Sans', sans-serif;
+
+ */
+body {
+    /* background-color: white;
+    background-color: hsla(102, 34%, 50%, 0.3); */
+    background-color: hsla(0, 100%, 71%, 0.2);
+    color: black;
+    max-width: 800px;
+    margin: 0 auto;
+}
+
+h1 {
+    text-align: center;
+    font-size: 57px;
+    font-family: 'PT Sans', sans-serif;
+    font-weight: bold;
+    letter-spacing: 2px;
+    margin: 40px 0 18px 0;
+    /* color: rgb(87, 2, 30); */
+
+}
+
+h2 {
+    font-size: 13px;
+    font-family: 'PT Sans', sans-serif;
+    text-align: center;
+    margin: 0 0 0 0;
```

You can shorten this to a single zero, since the measurements are the same for each side of the box:
```suggestion
    margin: 0;
```